README.md

```diff
@@ -1,21 +1,61 @@
-# javascript-utilities
-
-[![CircleCI](https://circleci.com/gh/Shopify/javascript-utilities.svg?style=svg)](https://circleci.com/gh/Shopify/javascript-utilities)
-
-A set of utilities for JavaScript and TypeScript projects at Shopify.
+<h1 align="center">
+  <br>
+  <a href="http://www.shopify.com/"><img src="https://cdn.shopify.com/assets2/press/brand/shopify-logo-main-small-f029fcaf14649a054509f6790ce2ce94d1f1c037b4015b4f106c5a67ab033f5b.png" alt="Shopify" width="200" /></a>
+  <br>
+  javascript-utilities
+  <br>
+</h1>
+
+<h4 align="center">A set of utilities for JavaScript and TypeScript projects at Shopify.</h4>
+
+<p align="center">
+  <a href="https://circleci.com/gh/Shopify/javascript-utilities">
+    <img src="https://circleci.com/gh/Shopify/javascript-utilities.svg?style=shield"
+         alt="CircleCI">
+  </a>
+  <a href="https://badge.fury.io/js/%40shopify%2Fjavascript-utilities">
+    <img src="https://badge.fury.io/js/%40shopify%2Fjavascript-utilities.svg" alt="npm version">
+  </a>
+  <a href="https://github.com/Shopify/javascript-utilities/blob/master/LICENSE.md">
+    <img src="https://img.shields.io/badge/License-MIT-green.svg" alt="License: MIT">
+  </a>
+</p>
+
+<div align="center">
```

Comment:
I'm indifferent, tbh, although I should become opinionated.

My thoughts with the current styling are that it helps, in general, to spot and fix typos when the text is on its own line.